packages/cli/src/utils/getWebpackConfig.ts

```diff
@@ -38,3 +39,23 @@ const getWebpackConfig = (entry: object, resolve: object, module: object) => ({
 });
 
 export default getWebpackConfig;
+
+function returnIfExists(path, continueFunction) {
+  try {
+    fs.accessSync(path, fs.constants.R_OK);
+    return path;
+  } catch (e) {
+    return continueFunction();
+  }
+}
+
+export function getWebpackConfigPath(executionPath) {
+  return returnIfExists(`${executionPath}/webpack.config.js`, () =>
```

Comment:
does this mean that you specify the folder and not the file itself in the configs? Shouldn't the complete path be configured? And in the default configs have the `webpack.config.js` in it?

Reply:
no its the file

Reply:
i tested it on testico, works like before